content/guides/visualization.md

```diff
@@ -16,6 +16,24 @@ NeuroData maintains a Neuroglancer [fork](https://github.com/neurodata/neuroglan
 1. Export and load state to/from an external JSON server
 1. Segmentation overlays for ARA and Zbrain ontologies, at [ara.viz.neurodata.io](https://ara.viz.neurodata.io) and [zbrain.viz.neurodata.io](https://zbrain.viz.neurodata.io)
 
+### View Local Images in Neuroglancer Example
+
+The following example can be used to help you view your own images in neuroglancer. This example uses a [script]([url('/content/guides/upload.py')]) to generate a test image, then convert from tif to precomputed format. It will also require running a [script](https://github.com/google/neuroglancer/blob/master/cors_webserver.py)  from neuroglancer to host the local data:
```

Comment:
The first link goes to a broken page when using a local build version of the site. Does the same thing happen when you build the site locally? You may have to give it it's own page or github location and replace this with a url link.

Reply:
I would also include information about:
- What versions of python it works with
- What python libraries are required to run (took several attempts at running before I had all libraries installed)

Reply:
What do you think of my changes? I told the user to install everything that's imported in upload.py.

Reply:
Regarding the link: are you able to infer which file I am trying to link to? If so, do you have any recs on how I should link it? I'm unsure because the file is not yet PR'ed in the repo so I don't know how I should link it.